pwa/src/templates/documentation/installationDocumentation/InstallationDocumentationTemplate.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import * as styles from "./InstallationDocumentationTemplate.module.css";
+import { Container } from "@conduction/components";
+import { Heading1, LeadParagraph, Link } from "@gemeente-denhaag/components-react";
+import { navigate } from "gatsby";
+import { ExternalLinkIcon } from "@gemeente-denhaag/icons";
+
+export const InstallationDocumentationTemplate: React.FC = () => {
+  return (
+    <Container layoutClassName={styles.container}>
+      <section className={styles.section}>
+        <Heading1>Installatie Documentatie</Heading1>
+
+        <LeadParagraph>
+          Praesent blandit laoreet nibh. Nullam accumsan lorem in dui. Praesent turpis. Nulla sit amet est. Nullam
+          accumsan lorem in dui. Ut varius tincidunt libero.{" "}
+          <span
+            onClick={() => {
+              navigate("#");
+            }}
+          >
+            <Link icon={<ExternalLinkIcon />} iconAlign="start">
+              Lees meer.
+            </Link>
+          </span>{" "}
+          Aenean commodo ligula eget dolor.
+        </LeadParagraph>
```

Comment:
The content for this is available [here](https://github.com/OpenCatalogi/web-app/blob/e3fdf396cd5fb39266fd77a2af404cb59a881cb7/README.md).

You can translate the texts and add them here. 

Note: there's a reference to the "Use this template" button within the text, which will not be available on the frontend (the button being referred to is the button on Github, but we're reading this on opencatalogi.nl). Therefore, we will need to re-phrase that a little bit. E.g. "Klik op de groene _Gebruik dit template_ op de [GitHub pagina](#)"